packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.tsx

```diff
@@ -99,7 +99,7 @@ export class AtomicSegmentedFacet
     };
     this.facet = buildFacet(this.bindings.engine, {options});
     this.facetId = this.facet.state.facetId;
-    registerFacetToStore(this.bindings.store, 'facets', {
+    registerFacetToStore(this.bindings.store, 'segmentedFacets', {
```

Comment:
Is there any purpose to registering segmented facets in the store?

Reply:
Yep it does not seem to serve any purpose as of now. Means all the code changes in `store.ts` for this PR could be removed at the same time.

Reply:
Actually, I did decide to register the facets in the store and added a field in the FacetInfo interface to be able to filter out facets that have breadcrumbs vs that don't